doc/docker/jenkinsnode/run-nice

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+set -ex
+ionice -c 3 nice $*
```

Comment:
Actually we do not have to be nice on this server, we are the only one there.

Reply:
Still, does it hurt to have it? if nothing else occupies resources the lower priority shouldn't really matter?

Reply:
I am not sure about that. The class we use is called "Idle".

Reply:
Ok I'll remove it for the v2 then, but i'll keep it in the readme as its meant to be general and add how to remove it as well if you don't want to use it while still working with out build jobs.